### PR TITLE
feat: Add support for Azure Blob storage

### PR DESCRIPTION
**Pull request by simster7**
* [X] Should we create container if it doesn't exist? Answer: No
* [ ] Better error handling
* [ ] Do we support directories?

### CODE REVIEW

File: api/openapi-spec/swagger.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "info": {
     "description": "Argo",
     "title": "Argo",
-    "version": "latest"
+    "version": "5043bf1a"
```
Comment (alexec) [inline api/openapi-spec/swagger.json:15]: error?

**Comment by simster7** [comment]
Ice-boxing this

**Comment by jaimeamesti** [comment]
Hi @simster7,

Why this PR was not completed?

**Comment by agilgur5** [comment]
This was eventually superseded by https://github.com/argoproj/argo-workflows/pull/9026

**Pull request closed without merging**